Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class DogFoodBowl : MonoBehaviourExtended
+{
+
```

Comment:
Our convention for PUP is to not include a newline before the initial open bracket of the class